web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx

```diff
@@ -171,6 +244,70 @@ export const MiniAppForm = (props: LinksFormProps) => {
           </p>
         )}
       </div>
+
+      <div className="grid gap-y-2">
+        <Typography variant={TYPOGRAPHY.H7}>Supported Countries</Typography>
+
+        <Typography variant={TYPOGRAPHY.R3} className="text-grey-500">
+          If your app is Mini App then you must select a list of countries where
```

Comment:
@andy-t-wang Please check this description. Let me know if we need to update it.